ExampleApp/AppDelegate.swift

```diff
@@ -234,3 +234,44 @@ class DummyMarketDataService: MarketDataService {
         onFailure(error)
     }
 }
+
+// Only implement this protocol if you need to stream your own market data
+class DummyStreamingMarketDataService: StreamingMarketDataService {
+    var timer: Timer?
+    var lastPrice = 2000.0
+    var onUpdate: ((TradeItQuote) -> Void)?
+    
+    @objc func updateQuote() {
+        let price = Double("\(arc4random_uniform(8000)).\(arc4random_uniform(99))")
+        
+        let quote = TradeItQuote()
+        quote.companyName = "Bitcoin"
+        quote.lastPrice = NSNumber(value: lastPrice)
+        quote.bidPrice = NSNumber(value: price!)
+        quote.askPrice = NSNumber(value: price!)
```

Comment:
Avoid force unwrapping even in dummy/example code. Partners might copy paste it.